recipe/meta.yaml

```diff
@@ -1,83 +1,52 @@
-{% set version = "2.9.4" %}
-{% set pfx = 'm2w64-' if win else '' %}
+{% set version = "3.0.4" %}
+{% set ver = version|replace(".", "_") %}
 
 package:
   name: rpy2
   version: {{version}}
 
 source:
-  url: https://pypi.io/packages/source/r/rpy2/rpy2-{{ version }}.tar.gz
-  sha256: be57f741d0c284b5d8785ab03dff0e829303e5ac30e548d5ceb46e05b168812e
-  patches:
```

Comment:
@mingwandroid none of these patches apply and I'm not sure if we should refresh them or if they were upstream. I guess that you were the author of those, can you give us a hand on which ones should we refresh/drop?